toolchain/build-toolchain.sh

```diff
@@ -1,69 +1,80 @@
-#!/bin/bash
+#!/usr/bin/env sh
 
-MAKE_OPTIONS="-j1"
-
-function download() { # $1=url, $2=file
-	if [ ! -f $2 ]; then
-		curl "$1$2" > "$2"
-	fi
-
-	if [ ! -d "${2%.tar.xz}" ]; then
-		tar xf "$2"
-	fi
+download() {
+	wget "$1" -O "$2"
```

Comment:
the `-O` option overwrites existing files which may be unnecessary. I think `wget --continue "$1"` would be cool though!

Reply:
That looks good to me! I've tested it and it all seems to be working correctly